sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,118 @@
+| `min_tag_size`                                | `double`              | `0.6`                            | The size of the AprilTag in meters (payload).                                                                                                             |
+| `max_tag_distance`                            | `double`              | `20.0`                           | Maximum allowed distance in meters from the camera to the tags.                                                                                           |
+| `max_allowed_homography_error`                | `double`              | `0.5`                            | AprilTag detection are discarded if the homography error is larger than `max_allowed_homography_error`.                                                   |
+| `use_receive_time`                            | `bool`                | `false`                          | Flag to determine whether to use the receive time instead of the header timestamps.                                                                       |
+| `use_rectified_image`                         | `bool`                | `true`                           | Flag to determine whether rectified images should be used in the calibration process.                                                                     |
+| `calibration_crossvalidation_training_ratio`  | `double`              | `0.7`                            | The ratio of data used for training versus testing during the calibration's cross-validation process.                                                     |
+| `calibration_convergence_min_pairs`           | `int`                 | `9`                              | The minimum number of AprilTag and LidarTag detection pairs required to consider the calibration process as potentially converged.                        |
+| `calibration_convergence_min_area_percentage` | `double`              | `0.005`                          | Minimum percentage of the area that needs to be covered by detection.                                                                                     |
+| `min_pnp_points`                              | `int`                 | `8`                              | Minimum number of points required for the Perspective-n-Point problem used in calibration to solve the pose estimation.                                   |
+| `min_convergence_time`                        | `double`              | `6.0`                            | Minimum time required for the calibration process to be considered as converged.                                                                          |
+| `max_no_observation_time`                     | `double`              | `3.0`                            | If the time between new detection and the last observed detection are larger than `max_no_observation_time`, remove the detection from active hypotheses. |
```

Comment:
The description for `max_no_observation_time` is wrong

Reply:
Sorry, could you tell me the correct explanation? Thanks

Reply:
Hypotheses can time out in two ways:
 - When a new observation comes in and the dt is higher than the threshold (same tag id)
 - When the latest observation and a particular observation have a dt higher than the threhold.
 
 In the past I used rclcpp::Time() to use the latest time, but there were errors in some use cases (at the time we still  had not decided on our policies regarding sim_time, so things broke a lot. Hence, I decided to use the latest detection stamp)